tests/callbacks_test.cpp

```diff
@@ -481,7 +533,7 @@ TEST_CASE("EarlyStopAtMinLossCallbackTest", "[CallbacksTest]")
   s.ExactObjective() = true;
 
   // The optimization process should return in one second.
-  const double result = s.Optimize(f, coordinates, EarlyStopAtMinLoss(100));
+  const double result = s.Optimize(f, coordinates, EarlyStopAtMinLoss<arma::mat>(100));
```

Comment:
```suggestion
  const double result = s.Optimize(f, coordinates, EarlyStopAtMinLoss(100));
```
(This is for reverse compatibility; once `EarlyStopAtMinLoss` is a type without template parameters, this change will be needed.)